app-admin/qtpass/qtpass-1.1.5.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+MY_P="QtPass-${PV}"
+
+inherit qmake-utils
+
+DESCRIPTION="multi-platform GUI for pass, the standard unix password manager"
+HOMEPAGE="https://qtpass.org/"
+SRC_URI="https://github.com/IJHack/${PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE="+qt5"
+DOCS=( FAQ.md README.md CONTRIBUTING.md )
+S="${WORKDIR}/${MY_P}"
+
+RDEPEND="qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5[xcb]
+		dev-qt/qtwidgets:5
+		dev-qt/qtnetwork:5
+	)
+	!qt5? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
```

Comment:
please remove Qt4 support if possible. Qt4 is deprecated, unsupported by upstream and we'd like to remove earlier than later if possible. https://wiki.gentoo.org/wiki/Project:Qt/Policies#Handling_different_versions_of_Qt